Homework1/Sorting/Sorting/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace Sorting
+{
+    class Program
```

Comment:
```suggestion
namespace Sorting;

﻿﻿﻿﻿using System;

class Program
```
namespace обычно пишется первой строкой файла, дальше точка с запятой, и тогда можно без лишних фигурных скобок и отступов.